src/Test.Utilities/AdditionalMetadataReferences.cs

```diff
@@ -45,6 +46,18 @@ public static class AdditionalMetadataReferences
                 new PackageIdentity("Microsoft.AspNetCore.Mvc", "1.1.8"),
                 new PackageIdentity("Microsoft.AspNetCore.Http", "1.1.2")));
 
+        public static ReferenceAssemblies DefaultWithNUnit { get; } = Default
+            .AddPackages(ImmutableArray.Create(new PackageIdentity("NUnit", "3.12.0")));
+
+        public static ReferenceAssemblies DefaultWithxunit { get; } = Default
```

Comment:
```suggestion
        public static ReferenceAssemblies DefaultWithXunit { get; } = Default
```

Reply:
Maybe use `DefaultWithXUnit` to keep the name consistent with `DefaultWithNUnit`?

Reply:
Fine by me. FYI I went with this name to follow the library naming convention.

Reply:
Ah, then it is okay to leave it as-is.